docs/osquery/osquery.asciidoc

```diff
@@ -82,23 +82,24 @@ You can run packs as live queries or schedule packs to run for one or more agent
 
 . Click the **Packs** tab.
 . Click **Add pack** to create a new pack, or click the name of an existing pack, then **Edit** to add queries to an existing pack.
+. Provide a name for the pack. The short description is optional. 
+. Schedule the pack to be deployed on specified agent policies (*Policy*) or on all agent policies (*Global*).
++
+If you choose the *Policy* option, you must also configure one or both of the following fields: 
++
+NOTE: When defining pack deployment details, you cannot configure the same policy multiple times. In other words, after specifying a policy, you can either choose to deploy the pack to all of the policy's agents or only a subset. You cannot choose both.
 
-. Provide the following fields:
-
-* The name of the pack.
-
-* A short description of the pack.
-
-* The agent policies where this pack should run. If no agent policies are set, the pack is not scheduled.
+** *Scheduled {agent} policies (optional)*: Allows you to deploy the pack to specific agent policies. By default, the pack is deployed to all {agents} that are registered to the policies you define.  
+** *Partial deployment (shards)*: Allows you to partially deploy the pack to specified agent policies. After defining a policy, use the *Shard* slider to limit the number of agents to which the pack is deployed. For example, after specifying a policy, you can choose to deploy the pack to half of the policy's agents by selecting 50% on the slider.
```

Comment:
'to limit the number of agents' - what do you think about changing number (I understand it as quantity) into something emphasizing '%' ?

Reply:
Sure, @tomsonpl! To make sure I'm understanding you correctly, are you asking that the word "number" be replaced with something less definite, such as "portion", "amount", or "subset"? For example:

 ```suggestion
** *Partial deployment (shards)*: Allows you to partially deploy the pack to specified agent policies. After defining a policy, use the *Shard* slider to set the amount of agents to which the pack is deployed. For example, after specifying a policy, you can choose to deploy the pack to half of the policy's agents by selecting 50% on the slider.
```

Reply:
Yes @nastasha-solomon. that was exactly my point, thank you:)  Do you think it makes sense?

Reply:
Yeah, I think using less definite terms to describe the sharding feature is a good call since users can't actually pick which agents will run the pack. They can only specify a percentage of the group.